pkg/operator/ceph/csi/template/rbd/csi-rbdplugin-provisioner-dep.yaml

```diff
@@ -81,13 +90,20 @@ spec:
             - "--leader-election-renew-deadline={{ .LeaderElectionRenewDeadline }}"
             - "--leader-election-retry-period={{ .LeaderElectionRetryPeriod }}"
             - "--default-fstype=ext4"
+            - "--logtostderr=false"
+            - "--alsologtostderr=true"
+            - "--log_file=/var/lib/cephcsi/log/rbdattacher.log"
```

Comment:
what happens if two provisioner runs on the same node? both will try to write to same log file. wont that cause any problem?

Reply:
I think wont be a problem, but I can test if needed.

Is there in general a possible usecase to run two provisioner on same node

Reply:
@Madhu-1 The provisioners currently enforce required anti-affinity, right?

Reply:
@travisn Yes correct its already set by Rook. we are good with that